### DISCUSSION TITLE
Tool Change Settings number

### DISCUSSION THREAD

**Discussion by siganberg** [grblHAL/plugins#28] [General]
Hi, 

Is there already existing setting number reserve for tool change parameters that I can use? 

Something like this to store all the configuration so user can easily adjust them:

https://github.com/rvalotta/rapidchange_atc/blob/6fe150f75e5ba4bfba8f51cdf81654bec761297b/my_plugin.c#L93

**Comment by terjeio** [comment]
Yes, I have reserved [this range](https://github.com/grblHAL/core/blob/0e55491ff876617ff293686a83ea6f8971944eb7/settings.h#L506-L508). Feel free to use it as you please as long as it is related to tool changers.

**Comment by siganberg** [comment]
Thanks!
